cmd/snap-repair/runner.go

```diff
@@ -121,6 +144,14 @@ func (r *Repair) Run() error {
 	// except the ones in "cmd.ExtraFiles" we are safe to set "3"
 	env = append(env, "SNAP_REPAIR_STATUS_FD=3")
 	env = append(env, "SNAP_REPAIR_RUN_DIR="+rundir)
+	// inject /usr/lib/snapd/ into PATH so that the script can use
+	// `snap-repair done`
+	for i, envStr := range env {
+		if strings.HasPrefix(envStr, "PATH=") {
+			newEnv := fmt.Sprintf("%s:%s", strings.TrimSuffix(envStr, ":"), repairToolsDir)
+			env[i] = newEnv
+		}
+	}
```

Comment:
Just for extra paranoia, would be nice to append a stock PATH here if we could not find one.